prov/gni/src/gnix_vc.c

```diff
@@ -1541,6 +1948,8 @@ static int __gnix_vc_connected(struct gnix_vc *vc)
 /* Schedule the VC for RX progress. */
 int _gnix_vc_rx_schedule(struct gnix_vc *vc)
 {
+	GNIX_TRACE(FI_LOG_EP_CTRL, "\n");
+
```

Comment:
i know its useful for debugging, but having ```GNIX_TRACE``` here can produce huge amounts of output.  May want to remove it.